src/participant/participant.ts

```diff
@@ -486,14 +489,17 @@ export default class ParticipantController {
       ![
         QUERY_GENERATION_STATE.ASK_FOR_DATABASE_NAME,
         QUERY_GENERATION_STATE.ASK_FOR_COLLECTION_NAME,
+        QUERY_GENERATION_STATE.CHANGE_DATABASE_NAME,
```

Comment:
To show the collections list to choose from, after we clicked on the different database name in the previously shown databases list.